modules/ROOT/pages/anypoint-monitoring.adoc

```diff
@@ -0,0 +1,187 @@
+= Anypoint Monitoring
```

Comment:
Might want to clarify that this is for PCE. "Anypoint Monitoring in PCE" maybe?

Reply:
Changed to Anypoint Monitoring on Anypoint Platform PCE